rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -45,9 +45,6 @@ trap anomaly_handler ERR INT TERM
 # - All versions : Check for the PostgreSQL version
 # - All versions : Create ncf technique update flag and restart at the end of the script
 # - 4.1.17       : Change LDAP attribute ipNetworkNumber -> networkSubnet
-# - 4.1.19, 4.3.9: Drop unused index nodeid_idx
-# - 4.1.21,4.3.10: Drop contraint rulesgroupjoin_pkey to replace it by an index
```

Comment:
4.1.19 was released in 01/2019 and 4.1.21 was released in 04.2019, so I guess these may still be necessary.

Reply:
oh I only saw 4.1 and not 4.3